plugin/storage/es/options.go

```diff
@@ -293,6 +310,17 @@ func (opt *Options) Get(namespace string) *config.Configuration {
 		nsCfg.servers = opt.primary.servers
 	}
 	nsCfg.Servers = strings.Split(nsCfg.servers, ",")
+
+	if nsCfg.remoteReadClusters == "" {
+		nsCfg.remoteReadClusters = opt.primary.remoteReadClusters
+	}
+
+	if nsCfg.remoteReadClusters == "" {
```

Comment:
can be omitted and inner block added to line 314

Reply:
Done